sql-db/sql-reactive/src/test/java/io/quarkus/ts/openshift/OpenShiftDB2IT.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.ts.openshift;
```

Comment:
I am not sure If Openshift test should have its own Packages, or share package with Reactive test.

Reply:
Well spotted! Yes, please rename the packages to rule out the openshift name.

Reply:
What's wrong with openshift name in package name? Personally, I would prefer to keep OCP and ordinary tests separately, to not make packages bigger than 5-7 classes.

Reply:
We are running the module on baremetal, with some optional OCP tests. That's why that having all the classes under `*.openshift.*` does not state the reality about what the module is being executed.
If you want to move the OpenShift tests into a `.openshift` package, fine with me.

Reply:
@Sgitario I am not sure if I understand you correctly. Tests which run on OCP are in package "io.quarkus.ts.openshift". Tests, which do not,  are in package "io.quarkus.ts.reactive". I would like to keep these tests in separate packages. Is this ok with you? If it's not, which package names would you prefer?

Reply:
It should be "io.quarkus.ts.reactive.openshift" or simply in "io.quarkus.ts.reactive".

Reply:
Changed.